subsys/bluetooth/mesh/pb_adv.c

```diff
@@ -557,6 +569,12 @@ static void send_reliable(void)
 			break;
 		}
 
+		if (BT_MESH_ADV(buf)->busy) {
+			continue;
+		}
+
+		BT_DBG("%u bytes: %s", buf->len, bt_hex(buf->data, buf->len));
```

Comment:
This should not be here will cause failed message should be part of the function below